aldryn_newsblog/models.py

```diff
@@ -76,8 +76,8 @@ class Article(TranslatableModel):
         search_data=models.TextField(blank=True, editable=False)
     )
 
-    content = PlaceholderField('aldryn_newsblog_article_content',
-                               related_name='aldryn_newsblog_articles',
+    content = PlaceholderField('newsblog_article_content',
```

Comment:
Not necessary. The placeholder shows up fine post-migration. However, the name of the placeholder as displayed in structure mode won't change until you save the article again.